test/parallel/test-repl.js

```diff
@@ -189,7 +189,15 @@ function error_test() {
     { client: client_unix, send: 'url.format("http://google.com")',
       expect: 'http://google.com/' },
     { client: client_unix, send: 'var path = 42; path',
-      expect: '42' }
+      expect: '42' },
+    // Regression for https://github.com/nodejs/io.js/issues/597
+    { client: client_unix, send: '.clear\nRegExp.$1\nRegExp.$2\nRegExp.$3\n' +
+                                 'RegExp.$4\nRegExp.$5\nRegExp.$6\n' +
+                                 'RegExp.$7\nRegExp.$8\nRegExp.$9\n',
+      expect: ['', '\'\'\n', '\'\'\n', '\'\'\n', '\'\'\n', '\'\'\n', '\'\'\n',
+               '\'\'\n', '\'\'\n', '\'\'\n'].join(`${prompt_unix}`) },
+    { client: client_unix, send: 'eval(RegExp.$1)',
+      expect: `undefined\n${prompt_unix}` },
```

Comment:
Even I thought so, that is why I included `.clear` in the first test. Shall I move these tests to the beginning?